trilogy-group/workbench-dify
# Concurrency stream fix

# Description

This adds support to run multiple agent chats concurrently.

## Type of Change


# How Has This Been Tested?

I have verified that the agents chats run concurrently & as expected for both of the following user flows:
1. go to studio tab -> open agent -> run app -> chat
2. go to explore tab -> open agent -> chat

# Suggested Checklist:
